src/transformers/backbone.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
This is not our copyright. I am not in favor of using a copy-pasted file for a base utility.